_data/simple-icons.json

```diff
@@ -7510,6 +7510,11 @@
                 ]
             }
         },
+        {
+            "title": "React Query",
+            "hex": "FF0000",
```

Comment:
How did you land on `#FF0000` (or `#111111`)? From the logo and website I would expect a color like either `#FF4154` or indeed `#002C4B` (taken from the logo) :thinking: 


> Shouldn't the color be one of the colors present in the logo, for example #002c4b?

The color should be the _brand_ color, it doesn't _have_ to be in the logo itself.